### PR TITLE
P0004 test

### PR DESCRIPTION
**Pull request by rahul1choudhary**
https://app.clickup.com/t/85ztwz7u4

### CODE REVIEW

**Review by rakeshshrivas — APPROVED** [review]

File: apps/proxy/src/app/users/user/user.component.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                                {{ element.mobile ?? '-' }}
+                            </ng-container>
+                        </td>
+                    </ng-container>
+
+                    <tr mat-header-row *matHeaderRowDef="displayedColumns; sticky: true"></tr>
+                    <tr
+                        mat-row
+                        *matRowDef="let row; columns: displayedColumns; let i = index"
+                        [ngClass]="{ 'last-child': i + 1 === pageSizeOptions[0] }"
```
Comment (prateek-giddh) [inline apps/proxy/src/app/users/user/user.component.html:76]: Avoid using hard coded index in `pageSizeOptions`

File: apps/proxy/src/app/features/create-feature/create-feature.component.html [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+                    }
+                "
+            ></ng-container>
+        </mat-error>
+    </div>
+</ng-template>
+
+<ng-template #matErrors let-label="label" let-fieldControl="fieldControl" let-patternErrorText="patternErrorText">
+    <mat-error *ngIf="fieldControl.errors?.required"> {{ label }} is required. </mat-error>
+    <mat-error *ngIf="fieldControl.errors?.minlengthWithSpace"> Min required length is 3 </mat-error>
```
[the remark this answers is not in the document]
Comment (rahul1choudhary) [reply apps/proxy/src/app/features/create-feature/create-feature.component.html:467]: it's `minLengthThreeWithoutSpace` which hard coded validator

File: apps/proxy/src/app/features/create-feature/create-feature.component.ts [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+            if (params?.id) {
+                this.featureId = params.id;
+                this.isEditMode = true;
+                this.getFeatureDetalis();
+            } else {
+                this.getFeatureType();
+            }
+        });
+        if (!this.isEditMode) {
+            this.featureType$.pipe(takeUntil(this.destroy$), filter(Boolean)).subscribe((features) => {
```
[the remark this answers is not in the document]
Comment (rahul1choudhary) [reply apps/proxy/src/app/features/create-feature/create-feature.component.ts:107]: done

File: apps/proxy/src/app/features/create-feature/create-feature.component.ts [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+        private componentStore: CreateFeatureComponentStore,
+        private activatedRoute: ActivatedRoute,
+        private toast: PrimeNgToastService,
+        private ngZone: NgZone
+    ) {
+        super();
+    }
+
+    ngOnInit(): void {
+        this.activatedRoute.params.subscribe((params) => {
```
[the remark this answers is not in the document]
Comment (rahul1choudhary) [reply apps/proxy/src/app/features/create-feature/create-feature.component.ts:97]: done

File: apps/proxy/src/app/users/user/user.component.html (hunk 1 above)
Comment (rahul1choudhary) [reply apps/proxy/src/app/users/user/user.component.html]: done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**